tools/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Instead of having a completely separate project for the x86 version, why not just compile ViewCompilation.csproj multiple times with different configurations? It will reduce the effort required to keep the two csproj files aligned.